tooling-mp-jwt-auth-tool/src/test/java/org/jboss/eap/qe/microprofile/jwt/auth/tool/EcKeyHelperTest.java

```diff
@@ -0,0 +1,28 @@
+package org.jboss.eap.qe.microprofile.jwt.auth.tool;
+
+import org.junit.Test;
+
+public class EcKeyHelperTest {
+
+    /**
+     * @tpTestDetails Reads a private key from a pem file
+     * @tpPassCrit Expect no Exceptions
+     * @tpSince EAP 7.4.0.CD19
+     */
+    @Test
+    public void readPrivateKey() throws Exception {
+        EcKeyHelper loader = new EcKeyHelper();
+        loader.readPrivateKey(EcKeyHelperTest.class.getClassLoader().getResource("pki/ES256/key.private.pkcs8.pem").toURI());
+    }
+
+    /**
+     * @tpTestDetails Reads a public key from a pem file
+     * @tpPassCrit Expect no Exceptions
+     * @tpSince EAP 7.4.0.CD19
```

Comment:
This was added later.

Reply:
@honza-kasik yes I added them last Friday

Reply:
@honza-kasik CD23 fixed